File: submissions/sm_029_sakshi/week_3/day_4/session_1/checksuperset.js
function checksuperset_arr(arr1,arr2)
{
    count=0
    if(arr2.length>arr1.length)
    {
        console.log("Not a superset");
    }
    else{
    for(var i=0;i<arr1.length;i++)
    {
        for(var j=0;j<arr2.length;j++)
        {
            if(arr1[i]==arr2[j])
            {
             count++
            }
            
        }
    }
    if(count<=arr1.length)
    {
        console.log("superset")
    }
       else{
           console.log("not superset")
       }
    }
    
}
checksuperset_arr([1,3,8,4,9,12],[1,3,89,45,76,88,9])

File: submissions/sm_103_apoorva/week_13/day_4/check_superset.py
li1 = set(map(int,input("Enter values of 1st set: ").split()))
li2 = set(map(int,input("Enter values of 2nd set: ").split()))
print(li1,li2)
count = 0
for i in li2:
    if i in li1:
        count += 1
print(count)
if count == len(li2):
    print("1st set is superset of 2nd")
else:
    print("1st set is not superset of 2nd")